### ISSUE TITLE
Add jobs in a view

### ISSUE THREAD

**Issue by CondormanFr** [jenkinsci/configuration-as-code-plugin#671]
Is there a way to add existing jobs inside a view in a jcasc file ?

```
 views:
  - list:
      name: "ViewName"
        jobNames:
            job: "Name of job1"
            job: "Name of job2"
...
```

**Comment by fwilhe** [comment]
Found out the hard way that applying a config destroys the configuration of the build monitor plugin. Would be really good to have this destruction free, and ideally configurable via the plugin.

**Comment by uhafner** [comment]
See also #222

**Comment by stevezau** [comment]
Anyway to do this? It's preventing me from using config-as-code-plugin. :(

**Comment by timja** [comment]
it shouldn't stop you using the plugin, you may need to use groovy for this particular usecase or send a PR to the existing plugin to make it work

**Comment by stevezau** [comment]
@timja hmm ok. I think i just confused as i'm doing via helm. Can you point me in the right direction/docs?

**Comment by timja** [comment]
in the right direction for what part?
I have some view support working here:
https://github.com/hmcts/cnp-jenkins-config/blob/master/cac-prod.yml#L23

**Comment by stevezau** [comment]
I want to restrict views to selected jobs. I can't see how to do this.

You suggested a groovy script? Looking for example of how to exec a groovy script with helm/jenkins in values.yaml

**Comment by casz** [comment]
sadly even after #838 still requires some work for jobs to be exported on views. There is no getter for the name :cry:

**Comment by casz** [comment]
Created https://issues.jenkins-ci.org/browse/JENKINS-57121

**Comment by timja** [comment]
There's a PR for it here:
https://github.com/jenkinsci/jenkins/pull/3994

**Comment by timja** [comment]
Fixed in jenkins core, Released in 2.180 of jenkins
